src/services/item/test/fixtures/items.ts

```diff
@@ -168,12 +170,16 @@ export class ItemTestUtils {
     itemMembership: ItemMembership;
     packedItem: PackedItem;
   }> => {
-    const { item, member, permission, creator, parentItem } = options;
+    const { item, permission, creator, parentItem } = options;
+    let member = options.member;
     const newItem = await this.saveItem({
       item,
       actor: creator ?? member,
       parentItem,
     });
+    if (!member) {
+      member = await saveMember();
+    }
```

Comment:
Why should we save the member if not null? Also, do you know why the member can be undefined? I know it's not you who initially allowed undefined member...

Reply:
Maybe it's just because the actor type in the tests is `Member | null`.

Reply:
I've created a use for that :)